src/Provider/Province.php

```diff
@@ -94,7 +94,7 @@ public function data()
             ['id' => '64', 'code' => '81', 'name_th' => 'กระบี่', 'name_en' => 'Krabi', 'geography_id' => '6'],
             ['id' => '65', 'code' => '82', 'name_th' => 'พังงา', 'name_en' => 'Phangnga', 'geography_id' => '6'],
             ['id' => '66', 'code' => '83', 'name_th' => 'ภูเก็ต', 'name_en' => 'Phuket', 'geography_id' => '6'],
-            ['id' => '67', 'code' => '84', 'name_th' => 'สุราษฎร์ธานี', 'name_en' => 'Surat Thani', 'geography_id' => '6'],
+            ['id' => '67', 'code' => '84', 'name_th' => 'สุราษฎร์ธานี', 'name_en' => 'SuratThani', 'geography_id' => '6'],
```

Comment:
What's about ID 67?
Should we fix **"Surat Thani"** to **"SuratThani"** ?

Reply:
Both not wrong but in thai this name no have space between two word 😃